README.md

```diff
@@ -22,15 +22,26 @@ from baiji.serialization import json
 with open(filename, 'w') as f:
     json.dump(foo, f)
 with open(filename, 'r') as f:
-    foo = json.load(foo, f)
+    foo = json.load(f)
 ```
 
 ```py
 from baiji.serialization import json
-json.dump(filename)
```

Comment:
This one should be put back as `json.dump(foo, filename)`. While it clearly wasn't written right, it's actually one of the best features of `baiji-serialization`, that you can pass a filename and not have to stress about opening the file first.